src/scripts/main.js

```diff
@@ -357,4 +357,24 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard tbody');
```

Comment:
Ensure that the element with the class 'dashboard' and a 'tbody' exists in the DOM. If it doesn't, this line will cause a runtime error.